.rubocop.yml

```diff
@@ -1,6 +1,126 @@
-inherit_from: .rubocop_todo.yml
+
 AllCops:
   Include:
     - Rakefile
     - Gemfile
     - '*.gemspec'
+
+# Broadly speaking, test code gets a pass for most of the Metrics family.
+#
+# IMO test code is not the place get pedantic about class length,
+# method complexity, etc.  One should be encouraged to add more tests
+# with minimal friction, not forced to make a hard choice between
+# cutting tests or splitting up my test suites.
+#
+Metrics/ClassLength:
+  Max: 400
+  Exclude:
+    - 'test/**/*.rb'
```

Comment:
This is typically done by means of a separate `test/.rubocop.yml` file rather than an Exclude directive on the top level Rubocop file. That approach allows for a bit more flexibility - in addition to disabling cops, one can also enable additional cops or configure existing cops differently for those folders.

I don't consider it a merge blocker. But if we'd like to follow the Ruby community's conventions (which I realize is only sometimes the case) I'd suggest that change.